Program.cs

```diff
@@ -1,7 +1,33 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿internal class Program {
+   private static void Main (string[] args) {
+      int a = 1;
+      int z = 101;
+      Random r = new ();
+      int n = r.Next (a, z);

```

Comment:
You could make z = 100 and use z + 1 since you are referencing z in other output statements.